packages/core/src/network.js

```diff
@@ -141,11 +141,24 @@ export class Network {
     return this.#aborted.has(requestId);
   }
 
+  logNetworkRequests(filter) {
+    let msg = '';
+    let reqs = Array.from(this.#requests.values()).filter(filter).map(r => r.url);
+    msg += `\n\n  ${['Active requests:', ...reqs].join('\n  - ')}\n`;
+    return msg;
+  }
+
   // Throw a better network timeout error
   _throwTimeoutError(msg, filter = () => true) {
     if (this.log.shouldLog('debug')) {
-      let reqs = Array.from(this.#requests.values()).filter(filter).map(r => r.url);
-      msg += `\n\n  ${['Active requests:', ...reqs].join('\n  - ')}\n`;
+      msg += this.logNetworkRequests(filter);
+    }
+
+    if (process.env.PERCY_IGNORE_TIMEOUT_ERROR === 'true') {
```

Comment:
This change only ignores network idle timeouts. Not page load. That you need to handle in page.ja

Reply:
`page.js` calls the function `_throwTimeoutError` to throw the error so if its set that is also handled.